HST/citations/__init__.py

```diff
@@ -151,10 +151,15 @@ def write(self, filename):
             f.write('\n')
             f.write(', '.join(self.authors))
             f.write('\n')
-            f.write(', '.self.title)
-            f.write('\n')
-            f.write(self.abstract)
+            f.write(self.title)
             f.write('\n')
+            if isinstance(self.abstract, list):
```

Comment:
I think you only care if abstract is an iterable, right? So it could be a list, tuple, dictionary, whatever. In which case the modern way to do this is:
```
from collections.abs import Iterable
... isinstance(self.abstract, Iterable) ...
```